examples/eth1_del_all_route_rule.yml

```diff
@@ -0,0 +1,5 @@
+---
+interfaces:
+  - name: eth1
+route-rules:
```

Comment:
In the add example, also a route is added and seems to be needed. Shouldn't we also remove it here?

Reply:
Also, is it possible to only remove the rule that we added in the example?

Reply:
Sure. Will do.

Reply:
No. Only support full editing for now.